frontend/src/routes/SingleShowing/AdminAction.js

```diff
@@ -132,8 +134,14 @@ class AdminAction extends Component {
       adminMessage
     } = this.state;
 
+    const ButtonContainer = styled.div`
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-evenly;
```

Comment:
Browser support for `space-evenly` is almost non-existent:

![image](https://user-images.githubusercontent.com/817046/29004603-6324202a-7aca-11e7-82b0-56a1a2ca321f.png)

Maybe you meant `space-between`?